docs/src/composables/versions.ts

```diff
@@ -0,0 +1,28 @@
+import { ref, onMounted } from 'vue'
+export const versions = [
+  {text: 'stable', link: '/',},
+  {text: 'dev', link: '/dev/',},
+  {text: 'v0.0.3', link: '/v.0.0.3/'}
+]
+
+// shared data across instances so we load only once.
+// const versions = ref([])
+
+// const dataHost = 'https://github.com/LuxDL/DocumenterVitepress.jl/blob/gh-pages/'
+// const dataUrl = `${dataHost}/versions.js`
```

Comment:
Can we use this instead?

Reply:
This way it's synced with `versions.js`for all items. See e.g. https://geo.makie.org/versions.js

Reply:
😄 , how? the issue is getting that list automatically? maybe from the Julia side then, read, parse and include?

Reply:
We could get the list by e.g.:
```js
const dataHost = 'https://luxdl.github.io/DocumenterVitepress.jl'
const dataUrl = `${dataHost}/versions.js`
```

see e.g. https://luxdl.github.io/DocumenterVitepress.jl/versions.js

but I'm not sure how the list would be constructed - or how it should be structured!

Reply:
this the template: 
```
versions = [
  {text: 'stable', link: '/v0.0.3/',},
  {text: 'dev', link: '/dev/',},
  {text: 'v0.0.3', link: '/v0.0.3/'}
```

Reply:
That could work as a first pass.  My issue with it is that it creates a static version selector, meaning that people on docs of v0.4 could not access v0.6 through the selector.

If possible, the best thing to do is probably to somehow access `versions.js` in the browser and generate a selector on the client side...